include/grpc/impl/codegen/compression_types.h

```diff
@@ -58,9 +58,9 @@ typedef enum {
   GRPC_COMPRESS_NONE = 0,
   GRPC_COMPRESS_DEFLATE,
   GRPC_COMPRESS_GZIP,
+  GRPC_COMPRESS_SNAPPY,
```

Comment:
a question I had here: is the int value of an enum option like `GRPC_COMPRESS_STREAM_GZIP` considered a part of the public API?  I wasn't sure if it is a bad practice to insert a new enum value in the middle of possible values here like this, which causes the int value of `GRPC_COMPRESS_STREAM_GZIP` to move up by one. 

For instance https://github.com/grpc/grpc/blob/master/doc/compression_cookbook.md#disable-compression-algorithms mentions using these int values to compute a bitset to set channel arguments like GRPC_COMPRESSION_CHANNEL_ENABLED_ALGORITHMS_BITSET

Adding `GRPC_COMPRESS_SNAPPY` here allowed the implementation of `grpc_compression_bitset_to_message_bitset` etc to remain unchanged,